plugins/arDominionB5Plugin/modules/menu/templates/_browseMenuInstitution.php

```diff
@@ -0,0 +1,8 @@
+<div class="dropdown mb-2">
```

Comment:
I think you could use `d-grid` in this element to make the children take the full width.

Reply:
Thanks, it works!